src/containers/Secrets/Secrets.js

```diff
@@ -45,11 +48,11 @@ export /* istanbul ignore next */ class Secrets extends Component {
   constructor(props) {
     super(props);
     this.state = initialState;
-    this.handleCreateSecretClick = this.handleCreateSecretClick.bind(this);
   }
 
   componentDidMount() {
     this.props.fetchSecrets();
+    this.props.fetchServiceAccounts();
```

Comment:
do we need to re-fetch service accounts when the websocket reconnects in `componentDidUpdate`?

Reply:
don't really know for sure but ill add it just in case.